viewer/src/public/migration/CogniteModelBase.ts

```diff
@@ -12,6 +13,13 @@ export interface CogniteModelBase {
   readonly type: SupportedModelTypes;
   dispose(): void;
   getModelBoundingBox(outBbox?: THREE.Box3): THREE.Box3;
+  /**
+   * Retrieves the camera position and target stored for the model. Typically this
+   * is used to store a good starting position for a model. Returns `undefined` if there
+   * isn't any stored camera configuration for the model.
+   */
+  getCameraConfiguration(): CameraConfiguration | undefined;
```

Comment:
I'm a bit unsure if we should have any documentation in the base model at all (or contrary in implementations). I just don't like that you have to copy-paste things from an interface to its implementations N times. Maybe we just should document things only once here in the interface.

Reply:
Removing for now.